src/nodes/ImageView.vue

```diff
@@ -227,6 +229,7 @@ export default {
 			this.imageLoaded = false
 			this.loaded = true
 		}
+		img.src = this.imageUrl
```

Comment:
Why did you move this?
Is it to make sure `onload` is defined before the image is actually loaded?
I agree it's better but it works fine without moving it.
@mejo- @juliushaertl @max-nextcloud What do you think about that?

Reply:
This is due to a race condition, detailed in the issue #1827 

tldr: Assigning Image.src before Image.onload does not fire the event if the actual `load` event on the Image object happens before assigning the event handler function. (so between `onload=` and `src=`).

I understand that this is extremely unlikely to happen, maybe even impossible or really a tiny edge-case. However, better safe than pull out our hair when something like that happens.

Reply:
Makes sense 👍